packages/app/src/store/studio-store.ts

```diff
@@ -175,6 +175,11 @@ export const useStudioStore = defineStore('studioRecorder', {
       this.newTestLineNumber = undefined
     },
 
+    needsProtocolCleanup () {
+      // Protocol cleanup (page reload) is only needed if Studio has actually been used for recording
+      return this._hasStarted || this.logs.length > 0 || this._isStudioCreatedTest
```

Comment:
Is the check on `logs` needed? We don't display studio logs anymore.

Reply:
yeah that condition isn't needed... I changed it to `this.testId`